source_gen/.gitignore

```
@@ -0,0 +1,5 @@
+.dart_tool
+.packages
+.pub
+packages
+pubspec.lock
```

```suggestion
.dart_tool
pubspec.lock
```

This can be cleaner 😄